src/web/components/elements/CalloutBlockComponent.tsx

```diff
@@ -40,6 +40,15 @@ const summeryStyles = css`
         display: none;
     }
     outline: none;
+
```

Comment:
Hey @liywjl - could you take a look at these changes for me? I moved from using onClick handler to prevent default to using the `pointer-events` in css - I think the behaviour is the same but if you've got a sec to scan your eyes over, that'd be great!

Reply:
just tested it and it works great! 👍